tutorcodejail/templates/codejail/apps/config/tutor.py

```diff
@@ -1,10 +1,16 @@
 """Module with the configuration of config classes."""
-from base import BaseConfig
+from codejailservice.config import DevelopmentConfig, ProductionConfig
 
 
-class DevConfig(BaseConfig):
-    """Class to use for development context that inherits from BaseConfig."""
+class DevelopmentConfig(DevelopmentConfig):
+    """Class to use for development context that inherits from DevelopmentConfig."""
 
+    {{patch("codejail-common-settings")}}
+    {{patch("codejail-development-settings")}}
```

Comment:
```suggestion
    {{ patch("codejail-common-settings") | indent(4) }}
    {{ patch("codejail-development-settings") | indent(4) }}
```

You will need the filter, otherwise it would be rendered outside the class. Same comment applies to `ProductionConfig`

Reply:
Hello Moi. Thanks for the feedback. But without the change you are proposing, I have the variable indent:
![image](https://github.com/eduNEXT/tutor-contrib-codejail/assets/35668326/488834b1-9f36-4a4c-ada3-d52402bb5a4f)
I will test what happens if I add your changes, but I think the settings will be over-indented.

Reply:
I tested using the indent, and it doesn't change the render of the code. I also tested with indent(8) and more, and the tutor.py file is not affected at all.

@MoisesGSalas

Reply:
I think it only shows if your patch is multiline, try something like:

```python
CODEJAIL_SETTINGS="""
CODE_JAIL["limits"]["REALTIME"] = 3
CODE_JAIL["limits"]["PROXY"] = True
"""


hooks.Filters.ENV_PATCHES.add_items(
    [
        ("codejail-common-settings", CODEJAIL_SETTINGS),
    ]
)
```

Reply:
You are right; I have already made the fix. Thanks.